doc/classes/Tree.xml

```diff
@@ -542,6 +545,9 @@
 		<theme_item name="children_hl_line_width" data_type="constant" type="int" default="1">
 			The width of the relationship lines between the selected [TreeItem] and its children.
 		</theme_item>
+		<theme_item name="dragging_unfold_wait_msec" data_type="constant" type="int" default="500">
+			During a drag-and-drop, this is how many milliseconds to wait over a section before the section unfolds.
```

Comment:
That's an odd thing for a theme property 🤔
I think it should be a regular property instead, but I suppose that'll make it more difficult to manage in the editor, so not sure.

Reply:
I made it a regular property at first. But that means every tree in the godot editor needs to get the property set, while also not setting it on trees in user space. So either all the contexts which create a tree need to set this to the config value, or we need a subclass of tree for the editor to use.
I figured doing it with a theme variable was more ergonomic, even though it does feel odd. But tree also has a theme property called `scroll_speed` so I felt ok about it.